apps/platform/Dockerfile

```diff
@@ -51,6 +51,7 @@ COPY --from=installer  --chown=nextjs:nodejs /app/apps/platform/public ./apps/pl
 ENV PORT=3025
 ENV NEXT_SHARP_PATH=/app/apps/platform/.next/sharp
 EXPOSE 3025
+# keyshade-ignore
```

Comment:
Why would we need these?

Reply:
this is for ks scan to ignore the ip address